app/routes.py

```diff
@@ -1,2 +1,74 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, abort, make_response, request
 
+planet_bp = Blueprint("planets", __name__, url_prefix = "/planets")
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message": f"{cls.__name__} {model_id} is invalid"}, 400))
+
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response({"message": f"{cls.__name__} {model_id} not found"}, 404))
+    
+    return model
```

Comment:
Great refactor to make this more generalizable to all models (even if they haven't been created yet)